MaterialComponents.podspec

```diff
@@ -1787,6 +1787,23 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "TextFields+Beta" do |extension|
```

Comment:
Tip: It would be nice if the extension name were a specific feature/project rather than the umbrella term "Beta."  That way we can have multiple things in Beta at any given time.

Reply:
Do any alternative extension names come to mind for you? I see your point about "Beta" not being ideal.

Reply:
Not being familiar with the project/goal, I'm not sure how to name it. For the Tabs work I'm doing, I'm planning to have multiple extensions for the same project (`+TabBarView` is current, `+NewTabBarViewController` (naming!) is planned).  I'm fine with `+Beta` if it's a "glob" target, otherwise "Naming is hard."

Reply:
Huh, I could rename it to `MaterialTextFields+ContainedInputViews`...